cfn.yaml

```diff
@@ -0,0 +1,136 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: handle outbound messages from salesforce to update zuora and identity
+Parameters:
+  Stack:
+    Description: Stack name
+    Type: String
+    Default: membership
+  App:
+    Description: Application name
+    Type: String
+    Default: salesforce-message-handler
+  Stage:
+    Description: Stage name
+    Type: String
+    AllowedValues:
+      - CODE
+      - PROD
+    Default: CODE
+  DeployBucket:
+    Description: Bucket where RiffRaff uploads artifacts on deploy
+    Type: String
+    Default: membership-dist
+Resources:
+  ExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action: sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: logs
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action:
+                - logs:CreateLogGroup
+                - logs:CreateLogStream
+                - logs:PutLogEvents
+              Resource: arn:aws:logs:*:*:*
+        - PolicyName: lambda
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action:
+                - lambda:InvokeFunction
+              Resource: "*"
+        - PolicyName: PrivateBucket
+          PolicyDocument:
+            Statement:
+               Effect: Allow
+               Action: s3:GetObject
+               Resource: !Sub arn:aws:s3:::membership-private/${Stage}/*
```

Comment:
needs the whole stage or just the $stage/salesforce-message-handler.private.conf ?